src/Genesis.sol

```diff
@@ -73,19 +81,19 @@ contract Genesis is ERC721, Ownable {
         override
         returns (string memory)
     {
-        if (!_exists(_tokenId)) revert TokenDoesNotExist();
+        if (!_exists(_tokenId))
+            revert TokenDoesNotExist();
 
-        if (genesisRenderer == address(0)) {
+        if (genesisRenderer == address(0))
             return "";
-        }
 
         MintInfo memory info = tokenToBlockNum[_tokenId];
 
         return
             IGenesisRenderer(genesisRenderer).tokenURI(
                 _tokenId,
-                info.blockNum,
-                info.blockdifficulty
+                info.blockNumber,
+                genesisMergeBlock
```

Comment:
this is suppose to be the block difficulty the minter minted at - for metadata purposes

Reply:
PoS doesn't have block.difficulty (it's prevrandao then) so I re-purposed it to show when genesisMergeBlock (since it isn't known while PoW, so it's a nice compromise)